.github/workflows/python.yml

```diff
@@ -203,3 +203,36 @@ jobs:
       - name: Test
         shell: bash
         run: ci/scripts/python_test.sh $(pwd) $(pwd)/build
+
+  manlinux1:
+    name: manlinux1 Python 2.4
+    runs-on: ubuntu-latest
+    if: ${{ !contains(github.event.pull_request.title, 'WIP') }}
+    env:
```

Comment:
These env variables are not required.

Reply:
I added them to be explicit about the build configuration. Since we don't have the dockerfile locally to inspect for the non-default config I think it's useful to repeat them here

Reply:
The issue is that these variables are not propagated to the wheel build script. For example `ARROW_BUILD_TYPE` will be set to `release` no matter the value here.

Reply:
Ah, I see. I'll remove them